### ISSUE TITLE
Use -frust-crate=

### ISSUE THREAD

**Issue by CohenArthur** [Rust-GCC/cargo-gccrs#56]
Since https://github.com/Rust-GCC/gccrs/pull/788 was merged, we can now set the crate name in gccrs via the cargo wrapper

**Comment by Taym95** [comment]
Hi @CohenArthur , will be nice to have doc on how to contribute to pick these first issues.

**Comment by CohenArthur** [comment]
Yeah sorry about that @Taym95 the project isn't really finished yet :) for this, you'll need to look at the `args.rs` file and understand how we structure GCC args vs rustc args. One of the steps will be to add handling for `--crate-name` and "translate" that to `-frust-crate` for the gccrs arguments.
